### PR TITLE
created a jframe and component and keyListeners

### PR DESCRIPTION
**Pull request by HelenSutton**

### CODE REVIEW

File: src/main/java/PinballFrame.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+    private void ballPressure() {
+        System.out.println("ball pressed");
+
+    }
+
+    @Override
+    public void keyPressed(KeyEvent e) {
+        int keyCode = e.getKeyCode();
+        if (keyCode == KeyEvent.VK_RIGHT){
```
Comment (esthergassner) [inline src/main/java/PinballFrame.java]: Should this be a switch statement?

File: src/main/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import org.jbox2d.common.*;
+import org.jbox2d.dynamics.*;
+
+public class Main {
+    public static void main(String[] args) {
+        Vec2 gravity = new Vec2(0.0f, -10.f);
+        World world = new World(gravity);
```
Comment (andrewoid) [inline src/main/java/Main.java]: This code is not used.

File: src/main/java/PinballKeyListeners.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+public class PinballKeyListeners implements KeyListener {
+    PinballController pinballController;
+
+    public PinballKeyListeners(PinballController controller) {
+        pinballController = controller;
+
+    }
+
+
```
Comment (andrewoid) [inline src/main/java/PinballKeyListeners.java]: delete all this extra whitespace

File: src/main/java/PinballComponent.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+
+public class PinballComponent extends JComponent {
+
+
```
Comment (andrewoid) [inline src/main/java/PinballComponent.java]: lots of extra white space

**Review by andrewoid — APPROVED** [review]

**Pull request merged**